medical_diagnostic_report/models/medical_observation.py

```diff
@@ -28,6 +28,11 @@ class MedicalObservation(models.Model):
         compute="_compute_interpretation",
     )
     # FHIR Field: interpretation
+    observation_date = fields.Datetime(
+        string="Date", related="diagnostic_report_id.issued_date"
```

Comment:
We need to store this value outside the relation, as an observation can be generated outside a report